components/Logo/Logo.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { Logo } from "./Logo";
+
+describe("Logo component", () => {
+  it("Renders blue logo by default", () => {
+    const { getByTestId } = render(<Logo />);
+    const logo = getByTestId("blue-logo");
+    expect(logo).toBeInTheDocument();
+  });
+
+  it("Renders white logo when darkBackground prop is true", () => {
+    const { getByTestId } = render(<Logo darkBackground={true} />);
+    const logo = getByTestId("white-logo");
+    expect(logo).toBeInTheDocument();
+  });
+
+  it("Should have a title", () => {
+    const { container } = render(<Logo />);
+    const title = container?.querySelector("svg")?.querySelector("title");
+    expect(title?.textContent).toEqual("Juguetear logo");
```

Comment:
Para buscar texto de accesibilidad, en lugar de usar el `container` para buscar un elemento con `querySelector`, podriamos usar el metodo [`getByRole`](https://testing-library.com/docs/queries/about#priority). Y podriamos esperar que el metodo `toHaveAccessibleName` contenga un texto accesible.

Por ejemplo:
```suggestion
    const { getByRole } = render(<Logo />)
    const title = getByRole("img", {
      name: /juguetear logo/i,
    })
    expect(title).toHaveAccessibleName("Juguetear logo")
```
Algo similar se podria aplicar en el test que busca el `role` y espera que esa `img`

Reply:
Bien, lo modifico. Esto es por algun tema de performance? o convenci'on?

Reply:
`Algo similar se podria aplicar en el test que busca el role y espera que esa img`
Para ese caso te referias a esto por ejemplo?:
`it("Should have a role of img", () => {
    const { getByRole } = render(<Logo />);
    const title = getByRole("img");
    expect(title).toBeInTheDocument();
  });`

Reply:
Asi es! el nombre `title` le queda un poco raro, pero el test esta bien!